platforms/hyperledger-besu/configuration/samples/network-besu.yaml

```diff
@@ -136,6 +139,9 @@ network:
             ambassador: 15013       #Port exposed on ambassador service (use one port per org if using single cluster)
           ws:
             port: 8546
+          metrics:
+            enabled: true
```

Comment:
I don't think we need 2 fields for metrics, if the port is absent, it should be taken as `disabled` by default.